includes/migration/wc/class-fs-wc-migration.php

```diff
@@ -12,44 +12,28 @@
 
 	class FS_WC_Migration extends FS_Migration_Abstract {
 
-		/**
-		 * @var EDD_Software_Licensing
-		 */
-		protected static $_edd_sl;
-
+		/** @var FS_WC_Migration_Endpoint Instance */
+		protected $ep;
```

Comment:
@shramee `FS_WC_Migration` and `FS_WC_Migration_Endpoint` were intentionally designed to be isolated classes. Based on reviewing your code, you are using `$ep` to access the order information. I recommend removing this dependency and just passing the minimal arguments that you need to access the order data (based on what I've seen it's the customer email and license key).